app/helpers/template_helpers.py

```diff
@@ -1,304 +1,378 @@
-        "crest": True,
-        "newTabWarning": lazy_gettext("The following links open in a new tab"),
-        "copyrightDeclaration": {
-            "copyright": lazy_gettext(
-                "Crown copyright and database rights 2021 NIMA MOU577.501."
-            )
-            if theme == "census-nisra"
-            else lazy_gettext("Crown copyright and database rights 2020 OS 100019153."),
-            "text": lazy_gettext(
-                "Use of address data is subject to the terms and conditions."
+    @property
+    def context(self) -> dict[str, Any]:
+        return {
+            "account_service_url": self._account_service_url,
+            "account_service_log_out_url": self._account_service_log_out_url,
+            "contact_us_url": Link(
+                "Contact us", f"{self.context_options.base_url}/contact-us/"
+            ).__dict__,
+            "cookie_settings_url": self._cookie_settings_url,
+            "page_header": self.page_header_context,
+            "footer": self.footer_context,
+            "languages": get_languages_context(self._language),
+            "theme": self.context_options.design_system_theme,
+            "language_code": self._language,
+            "schema_theme": self._theme,
+            "survey_title": self.context_options.survey_title,
+            "cdn_url": self._cdn_url,
+            "address_lookup_api_url": self._address_lookup_api,
+            "data_layer": self.context_options.data_layer,
+            "include_csrf_token": (
+                request.url_rule and "POST" in request.url_rule.methods
             ),
-        },
-        "rows": [
-            {"itemsList": items_list},
-        ],
-        "legal": [
-            {
-                "itemsList": [
-                    {
-                        "text": lazy_gettext("Cookies"),
-                        "url": static_content_urls["cookies"],
-                        "target": "_blank",
-                    },
-                    {
-                        "text": lazy_gettext("Accessibility statement"),
-                        "url": static_content_urls["accessibility_statement"],
-                        "target": "_blank",
-                    },
-                    {
-                        "text": lazy_gettext("Privacy and data protection"),
-                        "url": static_content_urls["privacy_and_data_protection"],
-                        "target": "_blank",
-                    },
-                    {
-                        "text": lazy_gettext("Terms and conditions"),
-                        "url": static_content_urls["terms_and_conditions"],
-                        "target": "_blank",
-                    },
-                ],
-            },
-        ],
-    }
+            "google_tag_manager_id": self._google_tag_manager_id,
+            "google_tag_manager_auth": self._google_tag_manager_auth,
+        }
+
+    @property
+    def page_header_context(self) -> dict[str, Any]:
```

Comment:
The header background seems to be misconfigured. For census, it is no longer purple:
![image](https://user-images.githubusercontent.com/35296336/120692408-a1e64900-c49f-11eb-810d-d2203f191a71.png)

Reply:
As discussed IRL, this was a result of an incorrect assignment in the dataclass, which has now been fixed